user-guide/Standard_Apps/TAG/Installing_TAG_App.md

```diff
@@ -0,0 +1,81 @@
+---
+uid: Installing_TAG_App
+---
+
+# Installing the DataMiner TAG Management app
+
+## Prerequisites
+
+- DataMiner version 10.4.2 or higher
+
+- A connection to a TAG MCS system
+
+- The DataMiner TAG Management app contains multiple functions that allows interaction with the TAG MCS or TAG MCM elements available in the system. To ensure the app works properly, it is necessary to have the following versions set to production at minimum:
+
+  - TAG Video Systems MCM-9000 connector version 1.1.6.10
+
+  - TAG Video Systems Media Control System (MCS) connector version 1.0.2.1
+
+  > [!NOTE]
+  > When you download the *TAG Management* package from the Catalog, these connectors are included and do not need to be installed separately.
+
+## Deploying the DataMiner TAG Management app
+
+If your system is **cloud-connected**:
+
+1. Look up the [*TAG Management* package](https://catalog.dataminer.services/details/package/6076) in the DataMiner Catalog.
+
+1. Click the *Deploy* button.
+
+1. Select the target DataMiner System and confirm the deployment. The package will be pushed to the DataMiner System.
+
+If your system is **not connected to the cloud**:
+
+1. Look up the [*TAG Management* package](https://catalog.dataminer.services/details/package/6076) in the DataMiner Catalog.
+
+1. Click the *Download* button.
+
+1. In DataMiner Cube, go to *Apps > System Center > Agents > Manage*, and install the package in the same manner as a [DataMiner upgrade](xref:Upgrading_a_DataMiner_Agent_in_System_Center).

```

Comment:
@sebastianulloa, could you double-check this step? I tried this out, and got the following error:

![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/92f43bc7-6eac-4266-80cb-a5f50f4d6fca)

Reply:
Modified steps to install locally as this were copy pasted from PTP package installation and it does not work the same.

Reply:
Even with these new steps, I cannot install the package after I've downloaded it. I am cloud-connected, so perhaps that is the issue? Could you look into this?
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/db1e55c3-83a6-4dac-88be-7ffa3ec998dd)